tests/pretty/issue-116799.rs

```diff
@@ -0,0 +1,10 @@
+// pretty-compare-only
+// pretty-mode:expanded
+// pp-exact:issue-116799.pp
```

Comment:
could you rename this test and the pp file to `format-args-str-escape` instead? (or something similar)

Reply:
Done.